packages/dd-trace/src/config.js

```diff
@@ -546,7 +547,7 @@ class Config {
     this._setValue(defaults, 'tracePropagationStyle.otelPropagators', false)
     this._setValue(defaults, 'tracing', true)
     this._setValue(defaults, 'url', undefined)
-    this._setValue(defaults, 'version', pkg.version)
```

Comment:
Is this a breaking change? What implications does it come with?

Reply:
Revert this and I think we can land it

Reply:
reverted